modules/healthconnectonfhir/src/test/java/edu/stanford/healthconnectonfhir/CreateObservationTests.kt

```diff
@@ -0,0 +1,254 @@
+package edu.stanford.healthconnectonfhir
+
+import androidx.health.connect.client.records.ActiveCaloriesBurnedRecord
+import androidx.health.connect.client.records.BloodPressureRecord
+import androidx.health.connect.client.records.BodyTemperatureRecord
+import androidx.health.connect.client.records.HeightRecord
+import androidx.health.connect.client.records.StepsRecord
+import androidx.health.connect.client.records.WeightRecord
+import androidx.health.connect.client.records.HeartRateRecord
+import androidx.health.connect.client.units.Energy
+import androidx.health.connect.client.units.Length
+import androidx.health.connect.client.units.Mass
+import androidx.health.connect.client.units.Pressure
+import androidx.health.connect.client.units.Temperature
+import ca.uhn.fhir.context.FhirContext
+import org.hl7.fhir.r4.model.Observation
+import org.hl7.fhir.r4.model.Period
+import org.hl7.fhir.r4.model.Quantity
+import org.junit.Test
+
+import org.junit.Assert.*
+import java.time.Instant
+import java.time.ZoneOffset
+import java.util.Date
+
+class CreateObservationTests {
+    private val fhirContext: FhirContext = FhirContext.forR4()
+
+    private fun printObservationAsJson(observation: Observation) {
+        val parser = fhirContext.newJsonParser()
+        val jsonString = parser.encodeResourceToString(observation)
+        println(jsonString)
```

Comment:
Thanks @eldcn, the `core:logging` module looks ideal and will definitely use it moving forward. In this case, the logging can actually be removed because it was just for visual inspection of the FHIR Observation's JSON representation but is not needed for the unit tests.